src/flick/comment/serializers.py

```diff
@@ -0,0 +1,22 @@
+from user.profile_simple_serializers import ProfileSimpleSerializer
+
+from rest_framework import serializers
+
+from .models import Comment
+
+
+class CommentSerializer(serializers.ModelSerializer):
+    comment_id = serializers.CharField(source="id", read_only=True)
```

Comment:
i used to do stuff like `tag_id` but now i think it might actually be better just to leave it as `id`